doc/source/whatsnew/v0.21.0.txt

```diff
@@ -235,6 +235,9 @@ Other Enhancements
 - Improved the import time of pandas by about 2.25x.  (:issue:`16764`)
 - :func:`read_json` and :func:`to_json` now accept a ``compression`` argument which allows them to transparently handle compressed files. (:issue:`17798`)
 - :func:`Series.reindex`, :func:`DataFrame.reindex`, :func:`Index.get_indexer` now support list-like argument for ``tolerance``. (:issue:`17367`)
+- ``GroupBy`` objects now have a ``pipe`` method, similar to the one on ``DataFrame`` and ``Series``
+  that allow for functions that take a ``GroupBy`` to be composed in a clean, readable syntax.
+  See the :ref:`documentation <groupby.pipe>` for more. (:issue:`17871`)
```

Comment:
you can move this to highlites is ok

Reply:
Ok, I'll move this to highlights and add an example in the whatsnew (if I understand you correctly?)

Reply:
yep

Reply:
Done.